docs/src/userguide/config/event_config.rst

```diff
@@ -299,10 +299,48 @@ Example configuration for `Google BigQuery`_, by specifying a query in the BigQu
 
     Inherited from :ref:`global event input config <skip-klio-read>`.
 
+
+.. _event-config-avro-read:
+
+Avro
+^^^^
+
+*Mode: Batch*
+
+``KlioReadFromAvro`` reads records in from the provided avro location.
+If the records' schema include an ``element`` field,
+the KlioMessage ``data.element`` will be set to the value.
+If there is no ``element`` field on the records read in from the avro,
+the entire record will be cast to bytes and stuffed into
+the KlioMessage ``data.element`` field.
+
+Example configuration for reading elements from a file:
+
+.. code-block:: yaml
+
+    name: my-cool-batch-job
+    pipeline_options:
+        streaming: False
+    job_config:
+        events:
+            inputs:
+                - type: avro
+
+Example pipeline reading in elements from an avro file:
+
+.. code-block:: python
+
+    def run(pipeline, job_config):
+
+        initial_data_path = os.path.join(DIRECTORY_TO_AVRO, "twitter.avro")
+        pipeline | io_transforms.KlioReadFromAvro(
+                file_pattern=initial_data_path
+        ) | beam.ParDo(transforms.HelloKlio())
+
+
```

Comment:
The other IOs on this page actually document the supported configuration key/values. Can you add them here too (for the output as well)? I would expect something like:

```rst
.. option:: job_config.events.inputs[].type

    Value: ``avro``

    | **Runner**: Dataflow, Direct
    | *Required*

.. option:: job_config.events.inputs[].file_pattern STR

    The file glob to read.

    | **Runner**: Dataflow, Direct
    | *Required*

etc...
```

There should be an entry for [all available config values](https://github.com/spotify/klio/blob/master/core/src/klio_core/config/_io.py#L341-L344).